.github/ISSUE_TEMPLATE/regular-issue.yaml

```diff
@@ -0,0 +1,47 @@
+name: Generic Issue
+description: Open a generic Issue for the OpenJS CPC
+labels:
+  - cross-project-council-agenda
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### OpenJS Cross Project Council
+
+        Thank you for opening an issue for the OpenJS Cross Project Council.
+
+        **Please remember to follow the Code of Conduct and be respectful.**
```

Comment:
I'm a net -1 on reminder people to behave until we have a track record of this being necessary.

Reply:
It's not a reminder to "hey behave", it's a "hey welcome to this community, these are the rules of this community in case you don't know'em"